docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md

```diff
@@ -4,13 +4,13 @@ author: Jayanth Kumar
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [ ] Install and configure Grafana for infrastructure metrics monitoring on EC2.
+- [ ] Set up data collection for CPU, RAM, and Disk usage metrics.
+- [ ] Design and create interactive dashboards for visualizing the metrics.
+- [ ] Document the entire configuration and dashboard setup process.
 
-## Screenshots / Videos 
-
-## Contributions
```

Comment:
Also what about contributions? Why haven't you added the contributions?

Reply:
My work is completely about setting up services on the aws ec2 instances.